bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/util/TestUtil.java

```diff
@@ -0,0 +1,37 @@
+package com.google.cloud.bigtable.util;
+
+import com.google.cloud.bigtable.config.Logger;
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class TestUtil {
```

Comment:
I haven't thought about the right package/ module to put this in yet. Putting it in this location for now just to show the code